pkg/models/vendor_verify_info.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2018 The OpenPitrix Authors. All rights reserved.
+// Use of this source code is governed by a Apache license
+// that can be found in the LICENSE file.
+
+package models
+
+import (
+	"context"
+	"time"
+
+	"openpitrix.io/openpitrix/pkg/logger"
+
+	"openpitrix.io/openpitrix/pkg/pb"
+	"openpitrix.io/openpitrix/pkg/util/pbutil"
+)
+
+type AppVendor struct {
+	UserId            string
+	CompanyName       string
+	CompanyWebsite    string
+	CompanyProfile    string
+	AuthorizerName    string
+	AuthorizerEmail   string
+	AuthorizerPhone   string
+	BankName          string
+	BankAccountName   string
+	BankAccountNumber string
+	Status            string
+	RejectMessage     string
+	SubmitTime        *time.Time
+	StatusTime        time.Time
+	UpdateTime        time.Time
+}
+
+func (vendor *AppVendor) ParseInputParams2Obj(req *pb.SubmitVendorVerifyInfoRequest) *AppVendor {
+	Vendor := AppVendor{}
+
+	//Vendor.UserId = req.GetUserId()
+	Vendor.UserId = req.GetUserId().GetValue()
+	Vendor.CompanyName = req.GetCompanyName().GetValue()
+	Vendor.CompanyWebsite = req.GetCompanyWebsite().GetValue()
+	Vendor.CompanyProfile = req.GetCompanyProfile().GetValue()
+	Vendor.AuthorizerName = req.GetAuthorizerName().GetValue()
+	Vendor.AuthorizerEmail = req.GetAuthorizerEmail().GetValue()
+	Vendor.AuthorizerPhone = req.GetAuthorizerPhone().GetValue()
+	Vendor.BankName = req.GetBankName().GetValue()
+	Vendor.BankAccountName = req.GetBankAccountName().GetValue()
+	Vendor.BankAccountNumber = req.GetBankAccountNumber().GetValue()
+	Vendor.Status = "submitted"
+	t := time.Now()
+	Vendor.SubmitTime = &t
+	Vendor.UpdateTime = time.Now()
+	Vendor.StatusTime = time.Now()
+	return &Vendor
+}
+
+func (vendor *AppVendor) ParseVendorSet2pbSet(ctx context.Context, invendors []*AppVendor) []*pb.VendorVerifyInfo {
```

Comment:
Change the func name to ParseVendorSet2PbSet

Reply:
If `invendors` is not one word, change to inVendors will be better.

Reply:
ok